cc/dual_net/tf_dual_net.cc

```diff
@@ -33,80 +37,79 @@ using tensorflow::TensorShape;
 
 namespace minigo {
 
-TfDualNet::TfDualNet(const std::string& graph_path) : graph_path_(graph_path) {
-  GraphDef graph_def;
-
-  // If we can't find the specified graph, try adding a .pb extension.
-  auto* env = Env::Default();
-  if (!env->FileExists(graph_path_).ok()) {
-    auto alt_path = absl::StrCat(graph_path_, ".pb");
-    if (env->FileExists(alt_path).ok()) {
-      std::cerr << graph_path << " doesn't exist, using " << alt_path
-                << std::endl;
-      graph_path_ = alt_path;
+namespace {
+class TfDualNet : public DualNet {
```

Comment:
This is addressed in PR #452. I did not keep any declarations in inference_server.h because it would be too different, and would prefer to follow up with a change that removes the other declarations from the header files.